week-2/interpret/time-format.js

```diff
@@ -19,25 +19,23 @@ function formatTimeDisplay(seconds) {
 
 console.log(formatTimeDisplay(143));
 
-// You can play computer with this example
-// Use the Python Visualiser https://pythontutor.com/visualize.html#mode=edit
-// to help you answer these questions
-
-// Questions
-
 // a) When formatTimeDisplay is called how many times will pad be called?
-
+//3 time
 // Call formatTimeDisplay with an input of 143, now answer the following:
 
 // b) What value is assigned to the parameter num when pad is called for the first time?
-
+   //remainingHours or 00
```

Comment:
00 would be the returned value of the method as you've correctly identified below. What value will be assigned to the _parameter_ when the function is called?